e2e/test/iothub/SasCredentialAuthenticationTests.cs

```diff
@@ -129,16 +129,15 @@ public async Task DigitalTwinClient_Http_SasCredentialAuth_Success()
             await deviceClient.OpenAsync().ConfigureAwait(false);
 
             string signature = TestConfiguration.IoTHub.GetIotHubSharedAccessSignature(TimeSpan.FromHours(1));
-            using var digitalTwinClient = DigitalTwinClient.Create(
+            using var serviceClient = new IotHubServiceClient(
                 TestConfiguration.IoTHub.GetIotHubHostName(),
                 new AzureSasCredential(signature));
 
             // act
-            HttpOperationResponse<ThermostatTwin, DigitalTwinGetHeaders> response = await digitalTwinClient
-                .GetDigitalTwinAsync<ThermostatTwin>(testDevice.Id)
+            DigitalTwinGetResponse<ThermostatTwin> response = await serviceClient.DigitalTwins
```

Comment:
What do you all think of this name? Would ```GetDigitalTwinResponse``` make more sense?

Reply:
It's been a while since I've looked at these APIs; do you have a sample response that shows what `DigitalTwinGetResponse<ThermostatTwin> response` looks like?

Reply:
Here is the definition for [GetDigitalTwinResponse](https://github.com/Azure/azure-iot-sdk-csharp/blob/timtay/digitaltwin/iothub/service/src/DigitalTwin/Models/GetDigitalTwinResponse.cs). The HTTP response from the server is defined such that the payload is the digital twin and there is a header that specifies the ETag for the digital twin

Reply:
In track2, a slight equivalent would be the request options for the calls (which we don't expose to the users, for the record).

<https://github.com/Azure/azure-sdk-for-net/blob/f35da107f6d8aa69ff3b0ac64fd3a1853f527212/sdk/digitaltwins/Azure.DigitalTwins.Core/src/Customized/Models/CreateOrReplaceDigitalTwinOptions.cs#L9>

In that case, we use the format: verb-noun-purpose.

So, I'd say GetDigitalTwinResponse would be consistent with that.

Reply:
I'll rename the request option types in this PR to match that pattern as well then